testing/ef_tests/src/case_result.rs

```diff
@@ -119,9 +119,13 @@ where
 }
 
 pub fn check_state_diff<T: EthSpec>(
-    pre_state: &BeaconState<T>,
-    opt_post_state: &Option<BeaconState<T>>,
+    _pre_state: &BeaconState<T>,
+    _opt_post_state: &Option<BeaconState<T>>,
 ) -> Result<(), Error> {
+    unimplemented!("`BeaconStateDiff` is not defined");
+    /*
+     * TODO(sproul): `BeaconStateDiff` is not defined.
```

Comment:
This was the old `milhouse` type. We could re-write this test using `HDiff` and `HDiffBuffer`, or just delete it until those names & interfaces have stabilized.

Reply:
Thanks, reimplemented with `HDiff` here: https://github.com/sigp/lighthouse/pull/4414/commits/a71b4123e7c24adbc50c885e71bebb9c4869ced3